CMakeLists.txt

```diff
@@ -68,10 +68,12 @@ if (DLAF_WITH_CUDA)
   find_package(CUDALIBS REQUIRED)
   set(DLAF_WITH_GPU ON)
 elseif(DLAF_WITH_HIP)
-  if (NOT ${CMAKE_CXX_COMPILER_ID} MATCHES "Clang")
-    message(FATAL_ERROR "DLAF_WITH_HIP=ON and the compiler used is not Clang. "
-      "Please specify CMAKE_CXX_COMPILER=hipcc or export CXX=hipcc.")
+  if (NOT CMAKE_HIP_ARCHITECTURES)
+    set(CMAKE_HIP_ARCHITECTURES "gfx900;gfx906" CACHE STRING "HIP architectures" FORCE)
```

Comment:
This is fine for testing, but should be removed before merging.

Reply:
Ugh, I didn't notice it's like that for CUDA. I'm in favour of removing it for CUDA as well, but perhaps someone else can comment if they think it should stay.